torchtitan/config_manager.py

```diff
@@ -212,6 +212,11 @@ def __init__(self):
             action="store_true",
             help="Whether to compile the model",
         )
+        self.parser.add_argument(
+            "--training.compile_rmsnorm",
+            action="store_true",
+            help="Whether to compile the norm layers",
```

Comment:
If it's called `compile_rmsnorm`, we should only compile if the `norm_type = "rmsnorm"`. The help message needs to reflect this.